invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/accounts_base.html

```diff
@@ -56,6 +56,12 @@
     {%- endblock head %}
   </head>
   <body ng-csp {% if body_css_classes %} class="{{ body_css_classes|join(' ') }}"{% endif %}{% if g.ln %} lang="{{ g.ln.split('_', 1)[0]|safe }}"{% if rtl_direction %} {{ rtl_direction|safe }}{% endif %}{% endif %} itemscope itemtype="http://schema.org/WebPage" data-spy="scroll" data-target=".scrollspy-target">
+    {% if not config.INVENIO_THEME_TUGRAZ_PRODUCTION %}
```

Comment:
We would likely be using the env variable to define these configs.
*Test* using ```env``` for defining the variable.
* create a ```.env``` file in your base instance directory , and assign the variable to true : ``` INVENIO_THEME_TUGRAZ_PRODUCTION=true```.
* test if it works.